components/infobox/commons/infobox_manufacturer.lua

```diff
@@ -0,0 +1,126 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Manufacturer
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Namespace = require('Module:Namespace')
+
+local BasicInfobox = Lua.import('Module:Infobox/Basic', {requireDevIfEnabled = true})
+local Locale = Lua.import('Module:Locale', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Header = Widgets.Header
+local Title = Widgets.Title
+local Center = Widgets.Center
+local Customizable = Widgets.Customizable
+local Builder = Widgets.Builder
+
+---@class ManufacturerInfobox: BasicInfobox
+local Manufacturer = Class.new(BasicInfobox)
+
+---@return Html
+function Manufacturer:createInfobox()
+	local infobox = self.infobox
+	local args = self.args
+
+	local widgets = {
+		Customizable{
+			id = 'header',
+			children = {
+				Header{
+					name = self:nameDisplay(args),
+					image = args.image,
+					imageDefault = args.default,
+					imageDark = args.imagedark or args.imagedarkmode,
+					imageDefaultDark = args.defaultdark or args.defaultdarkmode,
```

Comment:
Are the default images (placeholders) planned to be used?

Reply:
I don't imagine so, I'll remove them